src/wp-admin/css/login.css

```diff
@@ -27,14 +27,12 @@ a {
 
 a:hover,
 a:active {
-	color: #135e96;
+	color: var(--wp-admin--link--color--hover);
 }
 
 a:focus {
-	color: #043959;
-	box-shadow:
-		0 0 0 1px #4f94d4,
-		0 0 2px 1px rgba(79, 148, 212, 0.8);
+	color: var(--wp-admin--link--color--hover);
+	box-shadow: var(--wp-admin--box-shadow--focus);
```

Comment:
Are you doing just colors or entire box shadows?

Reply:
Still open to discussion, but in places where there are multiple box shadows, I think it makes sense to declare the whole value. For example, `--wp-admin--box-shadow--focus` is a pretty common box-shadow on focused items, and there is really no reason to control one of the shadow value but not the other, so for simplicity, it's one custom property.